packages/eth-deposit-to-different-address/scripts/exec.js

```diff
@@ -1,102 +1,112 @@
+  addCustomNetworkFromFile,
+} = require('arb-shared-dependencies')
 require('dotenv').config()
-requireEnvVariables(['DEVNET_PRIVKEY', 'L1RPC', 'L2RPC'])
+requireEnvVariables(['PRIVATE_KEY', 'CHAIN_RPC', 'PARENT_CHAIN_RPC'])
 
 /**
- * Set up: Connect to L1/L2 providers and instantiate an L1 wallet
+ * Set up: instantiate wallets connected to providers
  */
-const walletPrivateKey = process.env.DEVNET_PRIVKEY
+const walletPrivateKey = process.env.PRIVATE_KEY
 
-const l1Provider = new providers.JsonRpcProvider(process.env.L1RPC)
-const l2Provider = new providers.JsonRpcProvider(process.env.L2RPC)
+const parentChainProvider = new providers.JsonRpcProvider(
+  process.env.PARENT_CHAIN_RPC
+)
+const childChainProvider = new providers.JsonRpcProvider(process.env.CHAIN_RPC)
 
-const l1Wallet = new Wallet(walletPrivateKey, l1Provider)
+const parentChainWallet = new Wallet(walletPrivateKey, parentChainProvider)
+const childChainWallet = new Wallet(walletPrivateKey, childChainProvider)
 
 /**
- * Set the destination address and amount to be deposited in L2 (in wei)
+ * Set the destination address and amount to be deposited in the child chain (in wei)
  */
 const destAddress = '0x2D98cBc6f944c4bD36EdfE9f98cd7CB57faEC8d6'
-const ethToL2DepositAmount = parseEther('0.0001')
+const depositAmount = utils.parseEther('0.0001')
 
 const main = async () => {
   await arbLog('Deposit Eth via Arbitrum SDK on a different address')
 
   /**
-   * Add the default local network configuration to the SDK
-   * to allow this script to run on a local node
+   * Add the custom network configuration to the SDK if present
    */
-  addDefaultLocalNetwork()
+  addCustomNetworkFromFile()
 
   /**
-   * Use l2Network to create an Arbitrum SDK EthBridger instance
-   * We'll use EthBridger for its convenience methods around transferring ETH to L2
+   * Use childChainNetwork to create an Arbitrum SDK EthBridger instance
+   * We'll use EthBridger for its convenience methods around transferring ETH to the child chain
    */
-  const l2Network = await getL2Network(l2Provider)
-  const ethBridger = new EthBridger(l2Network)
+  const childChainNetwork = await getArbitrumNetwork(childChainProvider)
+  const ethBridger = new EthBridger(childChainNetwork)
 
   /**
    * First, let's check the ETH balance of the destination address
    */
-  const destinationAddressInitialEthBalance = await l2Provider.getBalance(
-    destAddress
-  )
+  const destinationAddressInitialEthBalance =
+    await childChainProvider.getBalance(destAddress)
 
   /**
-   * Transfer ether from L1 to L2
-   * This convenience method automatically queries for the retryable's max submission cost and forwards the appropriate amount to the specified address on L2
+   * Transfer ether from parent chain to a different address on child chain
+   * This convenience method automatically queries for the retryable's max submission cost and forwards the appropriate amount to the specified address on the child chain
```

Comment:
Can we add a sentence here that says `this tx will go through retryable ticket under hook`?

Reply:
Added this bit in the second sentence: `This convenience method automatically queries for the retryable's max submission cost and forwards the appropriate amount to the specified address on the child chain by using a retryable ticket instead of a regular deposit.`